Makefile

```diff
@@ -13,28 +15,37 @@ locale/downgrade.pot: downgrade
 		--package-name=downgrade \
 		--copyright-holder=$(AUTHOR) \
 		-o $@ $<
+	find ./locale/*po -exec msgmerge --update {} $@ \;
 
-downgrade.8: doc/downgrade.8.md
-	$(PANDOC) --standalone --to man doc/downgrade.8.md -o doc/downgrade.8
+doc/downgrade.8: doc/downgrade.8.md
+	$(PANDOC) --standalone --to man $< -o $@
 
-man: downgrade.8
+.PHONY: man
+man: doc/downgrade.8
 
+.PHONY: test
 test:
 	cram test
 
+.PHONY: install
 install:
-	install -Dm755 downgrade $(DESTDIR)/$(PREFIX)/bin/downgrade
-	install -Dm644 doc/downgrade.8 $(DESTDIR)/$(MANPREFIX)/man8/downgrade.8
-	install -Dm644 completion/bash $(DESTDIR)/$(PREFIX)/share/bash-completion/completions/downgrade
-	install -Dm644 completion/zsh $(DESTDIR)/$(PREFIX)/share/zsh/site-functions/_downgrade
+	install -Dm755 downgrade $(DESTDIR)$(PREFIX)/bin/downgrade
+	install -Dm644 doc/downgrade.8 $(DESTDIR)$(MANPREFIX)/man8/downgrade.8
+	install -Dm644 completion/bash $(DESTDIR)$(PREFIX)/share/bash-completion/completions/downgrade
+	install -Dm644 completion/zsh $(DESTDIR)$(PREFIX)/share/zsh/site-functions/_downgrade
+	for po_file in locale/*.po; do \
+	  locale="$$(basename "$$po_file" .po)"; \
+	  mkdir -p "$(DESTDIR)$(LOCALEPREFIX)/$$locale/LC_MESSAGES/"; \
+	  msgfmt "$$po_file" -o "$(DESTDIR)$(LOCALEPREFIX)/$$locale/LC_MESSAGES/downgrade.mo"; \
+	done
 
+.PHONY: uninstall
 uninstall:
-	$(RM) $(DESTDIR)/$(PREFIX)/bin/downgrade \
-	  $(DESTDIR)/$(MANPREFIX)/man8/downgrade.8 \
-	  $(DESTDIR)/$(PREFIX)/share/bash-completion/completions/downgrade \
-	  $(DESTDIR)/$(PREFIX)/share/zsh/site-functions/_downgrade
-
-.PHONY: setup test install uninstall
+	$(RM) $(DESTDIR)$(PREFIX)/bin/downgrade \
+	  $(DESTDIR)$(MANPREFIX)/man8/downgrade.8 \
+	  $(DESTDIR)$(PREFIX)/share/bash-completion/completions/downgrade \
+	  $(DESTDIR)$(PREFIX)/share/zsh/site-functions/_downgrade \
+	  $(DESTDIR)$(LOCALEPREFIX)/*/LC_MESSAGES/downgrade.mo
```

Comment:
This is a nice touch to combine all of the installation with locales in one place. Also my bad, I missed the uninstall issue for zsh completions.

Reply:
> A future commit will bring the locale files installation into the make install
target, which will respect PREFIX. That will only function with the
pre-requisite commit.

This sounds like a good addition. Maybe we could also bring man-doc building into that. But we can discuss that in a future PR.

Reply:
> Maybe we could also bring man-doc building into that. But we can discuss that in a future PR.

Agreed. My only concern would be bringing in a `pandoc` makedepend, it's a bit heavy.